frontend/ui/screensaver.lua

```diff
@@ -361,6 +365,7 @@ function Screensaver:show(event, fallback_message)
     end
     if screensaver_type == "message" then
         local screensaver_message = G_reader_settings:readSetting(prefix.."screensaver_message")
+        local messagePos = G_reader_settings:readSetting(prefix.."screensaver_message_position")
```

Comment:
```suggestion
        local message_pos = G_reader_settings:readSetting(prefix.."screensaver_message_position")
```
We use this shorthand to distinguish variables and functions. See https://github.com/koreader/koreader-base/wiki/Coding-style#naming